Dockerfile

```diff
@@ -117,8 +117,15 @@ RUN --mount=type=cache,target=/var/cache/apt --mount=type=cache,target=/var/lib/
 
 COPY --from=build /usr/local/src/ort/scripts/*.sh /opt/ort/bin/
 
+# This can be set to a folder containing crt-Files for custom certificates that ORT and all build tools should know about.
+ARG CA_FILES=""
```

Comment:
Are these really `CA` files, or `CRT` files?

Reply:
For us those are exclusively CA files, but if you use self-signed certificates, you'd have to actually include the certificate itself...